### ISSUE TITLE
Failed to execute mvn swagger:generate for generating swagger.json file

### ISSUE THREAD

**Issue by mccstan** [agoncal/agoncal-application-conference#18]
I am getting the following error when executing the second line, here a part of the output.
thnaks for helping.

[INFO] Building Conference 1.0
[INFO] ------------------------------------------------------------------------
[INFO] 
[INFO] --- swagger-maven-plugin:3.1.4:generate (default-cli) @ parent ---
[INFO] ------------------------------------------------------------------------
[INFO] Reactor Summary:
[INFO] 
[INFO] Conference ........................................ FAILURE [0.670s]
[INFO] Conference :: Commons ............................. SKIPPED
[INFO] Conference :: Commons JWT ......................... SKIPPED
[INFO] Conference :: Attendee ............................ SKIPPED
[INFO] Conference :: Rating .............................. SKIPPED
[INFO] Conference :: Tests ............................... SKIPPED
[INFO] Conference :: Schedule ............................ SKIPPED
[INFO] Conference :: Speaker ............................. SKIPPED
[INFO] Conference :: Talk ................................ SKIPPED
[INFO] Conference :: Venue ............................... SKIPPED
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 1.433s
[INFO] Finished at: Thu Dec 29 00:19:48 CET 2016
[INFO] Final Memory: 13M/130M
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal com.github.kongchen:swagger-maven-plugin:3.1.4:generate (default-cli) on project parent: You must configure at least one apiSources element -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoFailureException

**Comment by agoncal** [comment]
Yes, it looks like I can't use this goal on the parent. I have to have a better look at this.

In the meantime you can generate the swagger contract per sub-module : 

`mvn swagger:generate -f microservice-venue/pom.xml`
